src/pages/library/datasetBuilder/DatasetBuilderDetails.ts

```diff
@@ -0,0 +1,62 @@
+import { div, h, h1 } from 'react-hyperscript-helpers';
+import { ButtonOutline, spinnerOverlay } from 'src/components/common';
+import FooterWrapper from 'src/components/FooterWrapper';
+import { MarkdownViewer } from 'src/components/markdown';
+import TopBar from 'src/components/TopBar';
+import { DatasetBuilder, DatasetResponse } from 'src/libs/ajax/DatasetBuilder';
+import { useLoadedData } from 'src/libs/ajax/loaded-data/useLoadedData';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { useOnMount } from 'src/libs/react-utils';
+import { DatasetBuilderBreadcrumbs } from 'src/pages/library/datasetBuilder/Breadcrumbs';
+
+interface DatasetBuilderDetailsProps {
+  datasetId: string;
+}
+
+export const DatasetBuilderDetails = ({ datasetId }: DatasetBuilderDetailsProps) => {
+  const [datasetDetails, loadDatasetDetails] = useLoadedData<DatasetResponse>();
+
+  useOnMount(() => {
+    void loadDatasetDetails(() => DatasetBuilder().retrieveDataset(datasetId));
+  });
+  return datasetDetails.status === 'Ready'
+    ? h(FooterWrapper, [
+        h(TopBar, { title: 'Preview', href: '' }, []),
+        div({ style: { padding: '2rem' } }, [
+          h(DatasetBuilderBreadcrumbs, {
+            breadcrumbs: [{ link: Nav.getLink('library-datasets'), title: 'Data Browser' }],
+          }),
+          h1({ style: { marginTop: '0.75rem' } }, [datasetDetails.state.name]),
+          div({ style: { display: 'flex' } }, [
+            h(MarkdownViewer, [datasetDetails.state.description]),
```

Comment:
I was waiting on that content for your PR to merge, because I want to make use of some of the types in your PR/build on those. I could also merge your PR into mine now, but figured I'd do it one thing at a time.